test/release/examples/benchmarks/shootout/fannkuchredux.chpl

```diff
@@ -113,6 +113,7 @@ iter fannkuch(inds) {
 
 iter computeFact(n) {
   var f = 1;
+  yield f;
```

Comment:
It took me a little while to grok this. I would find it easier to follow if we did `const fact: [1..n] int = computeFact(n);`, but maybe we don't want that since it's the first thing you'd see when looking at the shootout?

Reply:
I also stumbled over this for a few minutes.

Reply:
Yeah, I agree it's subtle (I had to remember what the heck I was doing when reviewing it myself today), and would probably submit a version with the `: [1..n] int` as our replacement to the shootout site for clarity / backwards-compatibility.  I can change these versions as well for consistency if you prefer.

Reply:
Based on the reactions of the three of us, I implemented the explicit array form in https://github.com/chapel-lang/chapel/pull/14800/commits/870f734ef4b7bcc8471cbce75ecd98d8df0f3cd2

It's good motivation for finally implementing `var myArr: [1..] = ...` as a means of saying "I want this to be 1-based but don't want to bother telling you the upper bound and element type.